packages/shared/package.json

```diff
@@ -24,17 +24,17 @@
     "swr": "^1.3.0",
     "tsup": "^6.2.3",
     "typescript": "*",
-    "vitest": "^0.23.4"
-  },
-  "peerDependencies": {
+    "vitest": "^0.23.4",
```

Comment:
⛔ Vitest is a devDependency for this package and runs its tests. Adding it as a peerDependency won't work for NPM.